railties/lib/rails/secrets.rb

```diff
@@ -15,107 +15,110 @@ def initialize
     end
 
     @cipher = "aes-128-gcm"
-    @root = File # Wonky, but ensures `join` uses the current directory.
 
-    class << self
```

Comment:
I moved most of the class methods to instance ones, to ease of `path` use.
They were "nodoced" so I guess it doesn't require deprecation?